examples/inspec-windows-audit/habitat/default.toml

```diff
@@ -1,47 +1,54 @@
+
 #######################################
 # Optional settings
 #######################################
 
-# You don't usually need to change these.
-# This project provides highly tuned defaults for you.
-# If you don't have a strong reason for overriding these
-# Then it's a good idea to remove them.
+# You don't usually need to change these. This project provides highly tuned
+# defaults for you. If you don't have a strong reason for overriding these then
+# it's a good idea to remove them.
 
-interval = 1800
-# The number of seconds to wait between Chef Infra Client runs
+# The number of seconds to wait between Chef InSpec runs
 # Default value: 1800
+interval = 1800
 
-splay = 1800
-# A random number of seconds between 0 and $splay to add AFTER the interval. Use this setting to avoid the thundering herd problem. Optimal to always set this equal to the interval, or else you may end up DDOSing your own internal resources.
+# A random number of seconds between 0 and `splay` to add AFTER the interval.
+# Use this setting to avoid the thundering herd problem. It is recommended to
+# always set this equal to the interval, or else you may end up DDOSing your
+# own internal resources.
 # Default value: 1800
+splay = 1800
 
-splay_first_run = 0
-# A random number of seconds between 0 and $splay_first_run BEFORE the first Chef Infra Client runs, only on the first run. Used to avoid the thundering herd problem on new deployments. Optimal to always set to a function of your node count. Add 1 second for each node in your fleet. For example, if you have 1800 nodes, then you should set this value to 1800.
+# A random number of seconds between 0 and $splay_first_run BEFORE the first
+# Chef InSpec runs, only on the first run. Used to avoid the thundering herd
+# problem on new deployments. Optimally you should set this to a function of
+# your node count, adding 1 second for each node in your fleet. For example,
+# if you have 1800 nodes, then you should set this value to 1800.
 # Default value: 0
+splay_first_run = 0
 
-log_level = "warn"
-# The level of logging to be stored in a log file for the Chef Infra Client.
+# The level of logging to be stored in a log file for the Chef InSpec runs.
 # Possible levels: :auto (default), :debug, :info, :warn, :error, or :fatal.
-# Default value: :warn (when a terminal is available) or :info (when a terminal is not available).
+# Default value: :warn (when a terminal is available) or :info.
+log_level = "warn"
 
+# The connection information for your Chef Automate server/
+# Options:
+#  - enable: Set to `true` to enable, `false` to disable (Default: false)
+#  - server_url: Chef Automate server URL (Default: 'https://<automate_url>')
+#  - token: API token for Chef Automate (Default: '<automate_token>')
+#  - user: User for requests to Chef Automate (Default: '<automate_user>')
 [automate]
 enable = false
-server_url = "https://<automate_url>/data-collector/v0/"
+server_url = "https://<automate_url>"
```

Comment:
This is ideal, but currently is incorrect until we fix this issue. Please keep it as is.

Reply:
This needs the `/data-collector/v0` removed as the config has it in it [here](https://github.com/chef/effortless/blob/fcb877b39262b450ee6c00a28c8974239cb32400/scaffolding-chef-inspec/lib/scaffolding.ps1#L163)